src/js/api/set-config.js

```diff
@@ -6,57 +6,41 @@ const supportedFields = [
     'mute',
     'autostart'
 ];
-let model;
 
-function setVolume(mute, volume) {
-    const muteSet = !_.isUndefined(mute);
-    const volumeSet = !_.isUndefined(volume);
-
-    if (volumeSet) {
-        model.setVolume(volume);
-        if (!muteSet) {
-            model.setMute(volume === 0);
-        }
-    }
-
-    if (muteSet) {
-        model.setMute(mute);
-    }
```

Comment:
The reason we did this was to handle the situation where `setConfig` can handle the case when both `volume` and `mute` are set at the same time. Though unlikely, one use case I can think of is if the user wants to mute the audio, but have the volume set a specific value when the user chooses to unmute it, instead of what was set before it was muted. 

Without this code, the UI can get into a weird state where audio is playing but the mute icon is muted.